tests/gui/test_commands.py

```diff
@@ -89,6 +89,28 @@ def test_get_new_version_filename():
         PurePath("/a/b/labels.v02.slp")
     )
 
+def test_remove_videos_batch(
+        centered_pair_predictions: Labels,
+        small_robot_mp4_vid: Video,
+        centered_pair_vid: Video,
+        small_robot_3_frame_vid: Video,
+):
+    labels = centered_pair_predictions.copy()
+    labels.add_video(small_robot_mp4_vid)
+    labels.add_video(centered_pair_vid)
+    labels.add_video(small_robot_3_frame_vid)
+
+    all_videos = list(labels.videos)
+    assert len(all_videos) == 3
```

Comment:
Fix test

Reply:
Don't forget to resolve this conversation from the previous review if already addressed.